src/data/iconsets.ts

```diff
@@ -19,6 +19,8 @@ let toRead: Array<
   [string, (iconPath: string | undefined) => void, (e: any) => void]
 > = [];
 
+const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
```

Comment:
Export this from `src\util\is_safari.ts` and import here and in `src\components\ha-gauge.ts`